sdk/core/System.ClientModel/tests/client/ModelReaderWriter/ServiceModels/ApiProfile.Serialization.cs

```diff
@@ -3,8 +3,6 @@
 
 // <auto-generated/>
 
-#nullable disable
```

Comment:
These aren't needed one way or another because the `<auto-generated/>` comment opts-out of nullability annotations per: https://learn.microsoft.com/en-us/dotnet/csharp/nullable-migration-strategies